### DISCUSSION TITLE
STOP / Start after Update

### DISCUSSION THREAD

**Discussion by RadioHerzblut** [AzuraCast/AzuraCast#6723] [General]
Hello.

How do I terminate Azuracast with Putty.

How do I restart Azuracast with Putty?

Azuracast is on Ubuntu. I just did the update via Putty according to the instructions. Currently I still see the old version before the update ( v0.19.1 Stable - Docker - PHP 8.2 )

**Comment by BusterNeece** [comment]
@RadioHerzblut You should be able to run `./docker.sh restart` from `/var/azuracast` to restart our containers.
